### ISSUE TITLE
NOT filtering when searching or set as a template filter

### ISSUE THREAD

**Issue by iankoetter** [WowUp/WowUp#580]
**Is your feature request related to a problem? Please describe.**
I would like to be able to filter out results from both the Get Addons and My Addons views.

**Describe the solution you'd like**
Being able to filter things *out* of visibility would be nice. The query syntax could be `NOT TukUI` or `!TukUI` for example.

**Describe alternatives you've considered**
A checkbox to hide specific addons from view (with a 'Show/hide hidden results')

**Additional context**
Basically, add more options for filtering the results view. NOT with an option to save it eg: 

- [x] `always hide "ElvUI" and "TukUI"`

**Comment by jliddev** [comment]
This is already implemented in the latest Beta, via settings. Give it a shot.
